### PR TITLE
TransferWise Expense Flow Polishment

### PR DESCRIPTION
**Pull request by kewitz**
Requires https://github.com/opencollective/opencollective-api/pull/3587

### CODE REVIEW

File: components/expenses/PayoutBankInformationForm.js [hunk 1]
```diff
@@ -106,24 +106,29 @@ const RequiredFields = ({ disabled, getFieldName, formik, collective, currency }
...
         case 'select':
+          validate = value =>
+            input.valuesAllowed.some(v => v.key === value) ? undefined : 'This value is not accepted.';
           return (
-            <FastField key={input.key} name={fieldName}>
-              {({ field }) => (
-                <StyledInputField label={input.name}>
+            <FastField key={input.key} name={fieldName} validate={validate}>
+              {({ field, meta }) => (
+                <StyledInputField label={input.name} required error={meta.touched && meta.error}>
```
Comment (Betree) [inline components/expenses/PayoutBankInformationForm.js]: Why not using a `StyledSelect` to make sure only valid options can be picked?

Comment (kewitz) [reply components/expenses/PayoutBankInformationForm.js]: I'm not sure if I get it, the input child is a `StyledSelect` (line 135).

Comment (Betree) [reply components/expenses/PayoutBankInformationForm.js]: Indeed I missed that. Is the check above (`input.valuesAllowed.some(v => v.key === value) ? undefined : 'This value is not accepted.';`) still necessary?

Comment (kewitz) [reply components/expenses/PayoutBankInformationForm.js]: You're right, it is an overkill since I could just check if the value exists.

**Pull request merged**